app/src/main/java/io/multy/viewmodels/WalletViewModel.java

```diff
@@ -6,15 +6,18 @@
 
 package io.multy.viewmodels;
 
+import android.app.Activity;
```

Comment:
Remember rule "don't import android components inside viewmodel"

of course except .arch package